app/main/forms.py

```diff
@@ -74,13 +80,73 @@ class MoveUserForm(FlaskForm):
         ])
 
 
+class NotSuffixOfBlacklist:
```

Comment:
How about `class NotInDomainSuffixBlacklist`? (That matches our password blacklist validator naming more closely)

Reply:
Yyyyyyyeah I guess I was trying to make it more generic.